packages/cli/src/entrypoint.ts

```diff
@@ -4,11 +4,22 @@ import { Package } from "./package";
 import { JSONValue } from "./utils";
 import normalizePath from "normalize-path";
 
+export type ExportsConditions = {
+  module?: string;
+  default?: string;
```

Comment:
I would also seriously consider adding production/development conditions here. Webpack docs suggest that by default we should load "production" bundles and that development files should be opt-in (this would only require us to introduce development condition)

Reply:
I'm open to this... An example repo using prod/dev conditions would be really helpful. Everything I currently have was just following the conventions set by `vanilla-extract` repo.

Reply:
Examples of this can be found here:
https://webpack.js.org/guides/package-exports

As you can see - this stuff can get pretty crazy. I think we should assume "production" code as defaults~ and just add a development condition and point to dev files from there. This is what is suggested by webpack and I think it makes sense. It's would act as a progressive enhancement for dev environments configured to make use of those development files.